plugins/CoreHome/templates/_htmlEmailHeader.twig

```diff
@@ -0,0 +1,36 @@
+{% set fontStyle %}color:{{ themeStyles.colorText }};font-family:{{ themeStyles.fontFamilyBase }};{% endset %}
+
+<html style="background-color:#edecec">
+
+<head>
+    <meta charset="utf-8">
+    <meta name="robots" content="noindex,nofollow">
+    <meta name="generator" content="Matomo Analytics">
+</head>
+
+<body style="{{fontStyle}} line-height: 24px; margin:0 auto; max-width:1000px; background-color:rgb(255, 255, 255);box-shadow: 0px 0px 5px 1px rgba(0,0,0,0.75);">
+
+<a name="reportTop"></a>
+
+<table style="width:100%; background-color: {{ themeStyles.colorHeaderBackground }}; color: {{ themeStyles.colorHeaderText }}; padding:10px 0; margin: 0 0 25px 0; height:64px;">
+    <tr>
+        <td>
+            <a style="{{ fontStyle }}; font-size:16px;padding:0 15px;color: {{ themeStyles.colorHeaderText }};height: 22px;display: inline-block;vertical-align: inherit;" rel="noreferrer noopener" target="_blank" href="{{ piwikUrl }}" style="lineheight:17px">
```

Comment:
do we need to escape those attributes in a special way? like `fontStyle` etc? I presume it is fine since it is straight from the theme... so maybe even need to use `raw` filter? not sure...

Reply:
escaping w/ html_attr can't hurt.